mempool/src/core_mempool/transaction_store.rs

```diff
@@ -224,7 +224,10 @@ impl TransactionStore {
     pub(crate) fn insert(&mut self, txn: MempoolTransaction) -> MempoolStatus {
         let address = txn.get_sender();
         let txn_seq_num = txn.sequence_info.transaction_sequence_number;
-        let acc_seq_num = txn.sequence_info.account_sequence_number;
+        let acc_seq_num = max(
```

Comment:
This changes the behavior for all usages of acc_seq_num below, but most importantly for
L316 `self.process_ready_transactions(&address, acc_seq_num);`

It was still using the `txn.sequence_info.account_sequence_number` before this change.